### PR TITLE
Update README to reflect current project status

### PR DESCRIPTION
**Pull request by andogro**
### What Changed

- Text and linking cleanup
- Updated current sync progress with last updated date

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -157,10 +165,19 @@ The Ethereum common tests are in a submodule. To update you'll want to:
...
+| Encoding and Hashing                                                                | The [RLP] encoding protocol and the [Merkle Patricia Tree] data structure are fully implemented.                                                                                                                                                                                                                                                                                                                                                                                                       |
+| [Ethereum Virtual Machine] | Our EVM currently passes 100% of the common [VM tests]. We are still discovering subtle differences in our implementation such as the [“vanishing Ether” issue]. This component is for the most part complete.                                                                                                                                                                                                                                                       |
```
Comment (andogro) [inline README.md]: Is "vanishing Ether" still an issue? Any other additions to current project status welcome.

File: README.md [hunk 2]
```diff
@@ -2,16 +2,20 @@
 
 [![CircleCI](https://circleci.com/gh/mana-ethereum/mana.svg?style=svg)](https://circleci.com/gh/mana-ethereum/mana) [![Waffle.io - Columns and their card count](https://badge.waffle.io/mana-ethereum/mana.svg?columns=all)](https://waffle.io/mana-ethereum/mana)[![Gitter chat](https://badges.gitter.im/mana-ethereum/mana.png)](https://gitter.im/mana-ethereum/mana)
 
-Mana-Ethereum is an open-source Ethereum blockchain client built using [Elixir](https://elixir-lang.org/). Elixir runs on the Erlang Virtual Machine, which is used for distributed systems and offers massive scalability and high visibility. These properties make Elixir a perfect candidate for blockchain network development.
```
Comment (ayrat555) [inline README.md]: what did you remove links?

File: README.md [hunk 3]
```diff
@@ -2,16 +2,20 @@
...
+
+[Elixir]: https://elixir-lang.org/
+[Ethereum yellow paper]: https://ethereum.github.io/yellowpaper/paper.pdf
+[Project Status]: #project-status
 
 # Dependencies
 
 - Elixir ~> 1.7.3
-- Rust ~> 1.26.0 (as a dependency of [Rox](https://github.com/urbint/rox))
+- Rust ~> 1.26.0 (as a dependency of [Rox])
```
Comment (ayrat555) [inline README.md]: we should remove `rust` form dependencies. We don't use `rox` anymore

File: README.md [hunk 4]
```diff
@@ -25,11 +29,11 @@ git clone --recurse-submodules https://github.com/mana-ethereum/mana.git
 
 - Run `bin/setup`
 
+[Rox]: https://github.com/urbint/rox
```
Comment (ayrat555) [inline README.md]: this should be removed

File: README.md [hunk 5]
```diff
@@ -52,29 +56,31 @@ Alternatively, you can sync via IPC to a local node (like Parity or Geth running
...
 ### Known Sync Issues
+_Updated 11/6/2018_
 
-- When running the script mainnet fails due to an as of yet undiagnoised issue
-  (see [this issue]). The current highest block we've reached is [1660868].
-- Ropsten fails on block [295311] with `receipts_root_mismatch`,
-  `gas_used_mismatch` and `state_root_mismatch` errors.
+- Mainnet syncing slows considerably due to the [transaction spam attack] 
+that impacted Ethereum in late 2016. We are currently synced to ~ block [2469000] and counting.
+- Ropsten fails on block [2444903] with `receipts_root_mismatch`,
```
Comment (ayrat555) [inline README.md]: :+1:

File: README.md [hunk 6]
```diff
@@ -197,3 +216,4 @@ Additional Ethereum Implementations
 - [Go-Ethereum (Geth)](https://github.com/ethereum/go-ethereum/)
 - [EthereumJS](https://github.com/ethereumjs/ethereumjs-vm)
```
Comment (ayrat555) [inline README.md]: Maybe we should remove other implementations? Because we should be promoting out client

File: README.md (hunk 2 above)
Comment (andogro) [reply README.md]: I moved to the bottom of each section. Let me know if this is problematic and I can change back.

File: README.md (hunk 6 above)
Comment (andogro) [reply README.md]: Ok, I'm fine with removing.

File: README.md (hunk 3 above)
Comment (MattMSumner) [reply README.md]: Yup, this line can be removed 👍

File: README.md [hunk 7]
```diff
@@ -52,29 +56,31 @@ Alternatively, you can sync via IPC to a local node (like Parity or Geth running
...
 ```
 
-You can begin syncing, you will see a timestamp and a running list of verified blocks.
+Once syncing begins you will see a timestamp and a running list of verified blocks.
+
+[command-line interface]: https://github.com/mana-ethereum/mana/tree/master/apps/cli
+[Infura API key here]: https://infura.io/register
 
 ### Known Sync Issues
+_Updated 11/6/2018_
```
Comment (MattMSumner) [inline README.md]: Love this addition 👍

File: README.md (hunk 1 above)
Comment (MattMSumner) [reply README.md]: We've handled it so it's not an "issue" anymore. We may be able to remove this subtlety.

**Comment by andogro** [comment]
Submitting a revised pull request

**Pull request closed without merging**